classy-prelude/classy-prelude.cabal

```diff
@@ -1,5 +1,5 @@
 name:                classy-prelude
-version:             1.2.0.1
+version:             1.2.0.2
```

Comment:
This should be a bump to 1.3.0, since it can break existing code.

Reply:
And should classy-prelude-yesod/classy-prelude-conduit's versions change as well? I see that those have the same version number

Reply:
Updated PR accordingly